Formula/lgeneral.rb

```diff
@@ -16,10 +16,12 @@ class Lgeneral < Formula
     sha256 x86_64_linux:   "eefab1384276b2406cbdb286bfc730b3d434112f2d5ea8be6c3edb451d34f2e9"
   end
 
+  # setting deprecated, because brew audit fails build because or deprecated sdl_mixer
+  deprecate! date: "2023-02-24", because: :does_not_build
+
```

Comment:
This should not be deprecated unless the project is deprecated

Reply:
Then how to set it correctly to bypass audit fail ?
https://github.com/Homebrew/homebrew-core/pull/123219#issuecomment-1444164099

Reply:
It shouldn't depend on software that is deprecated. If it doesn't have sound without those, that's up to upstream to fix

Reply:
> I can raise a request to community to support sdl_mixer2, but i don't think they have a close plans to do that.

That is a good idea either way

> I would think that it is better to have sound with "deprecated" label (it is not far from reality because it would be better to update lgeneral code with direct SDL2 and sdl_mixer 2 support) rather than play in silence. 

If it was just old software I would agree. But the authors explicitly say you shouldn't use sdl_mixer anymore. So we don't want to add a dependency on that.